src/components/Article.jsx

```diff
@@ -1 +1,29 @@
-// export const Article = () => ();
+import React from 'react';
+
+export const Article = ({ title, text, date }) => {
+  const today = new Date(date);
+  const dateISO = today.toISOString().slice(0, 10);
+  const dateLocaleString = today.toLocaleDateString('en-US', {
```

Comment:
```suggestion
  const dateISO = date.toISOString().slice(0, 10);
  const dateLocaleString = date.toLocaleDateString('en-US', {
```
Date is already created, no need to duplicate it, just use the date that you receive in this component